Backend/lib/payment/src/Lib/Payment/Domain/Action.hs

```diff
@@ -53,13 +54,58 @@ createOrderService ::
   (Payment.CreateOrderReq -> m Payment.CreateOrderResp) ->
   m Payment.CreateOrderResp
 createOrderService merchantId personId orderId createOrderReq createOrderCall = do
-  existingOrder <- runInReplica $ QOrder.findById orderId
-  whenJust existingOrder $ \_ -> throwError (InvalidRequest "Payment order already exists")
-  createOrderResp <- createOrderCall createOrderReq -- api call
-  paymentOrder <- buildPaymentOrder merchantId personId orderId createOrderReq createOrderResp
-  Esq.runTransaction $
-    QOrder.create paymentOrder
-  pure createOrderResp
+  mbExistingOrder <- runInReplica $ QOrder.findById orderId
+  case mbExistingOrder of
+    Nothing -> do
+      createOrderResp <- createOrderCall createOrderReq -- api call
+      paymentOrder <- buildPaymentOrder merchantId personId orderId createOrderReq createOrderResp
+      Esq.runTransaction $
+        QOrder.create paymentOrder
+      pure createOrderResp
+    Just existingOrder -> do
+      sdk_payload <- buildSDKPayload createOrderReq existingOrder
+      pure
+        Payment.CreateOrderResp
+          { status = existingOrder.status,
+            id = "ordeh_xxxxxxxxxxxxxxxxxxxx", -- not saved in db
```

Comment:
we should save this ID in db, it is what juspay/payment_handler will generate at their end.

Reply:
ther is two different ids in ersponse:

```
  "order_id": "14183944763",
  "id": "ordeu_8bddad1e3b9c85eab3db4528575c01b",
```

First one it the same, as we send to Juspay payments, second one generated by Juspay payments server. You mean, that we need both?

Reply:
@piyushKumar-1 added this id also in db